packages/linter/src/rules/StoryRuntimeIsV1.ts

```diff
@@ -0,0 +1,13 @@
+import { Context } from "../index";
+import { Rule } from "../rule";
+
+export class StoryRuntimeIsV1 extends Rule {
+  run({ $ }: Context) {
+    const isV1 =
+      $("script[src='https://cdn.ampproject.org/v0/amp-story-1.0.js']").length >
```

Comment:
Sidenote: these rules will all break once custom runtime hosting will be supported.